charts/pega/config/deploy/catalina.properties

```diff
@@ -0,0 +1,219 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I am assuming we have taken the source code from tomcat container we run during deployments?

Reply:
Correct.  We picked the default file